test/commands/test/lib/included-files-parser-test.ts

```diff
@@ -1,7 +1,112 @@
 import { IFileDescriptionJson } from "../../../../src/commands/test/lib/test-manifest-reader";
 import { expect } from "chai";
-import { parseIncludedFiles } from "../../../../src/commands/test/lib/included-files-parser";
+import { copyIncludedFiles, parseIncludedFiles } from "../../../../src/commands/test/lib/included-files-parser";
+import * as xmlUtil from "../../../../src/commands/test/lib/xml-util";
+import * as pfs from "../../../../src/util/misc/promisfied-fs";
 import * as os from "os";
+import * as Sinon from "sinon";
+
+describe("copyIncludedFiles", () => {
+  let sandbox: Sinon.SinonSandbox = null;
+
+  beforeEach(() => {
+    sandbox = Sinon.sandbox.create();
+  });
+  afterEach(() => {
+      sandbox.restore();
+  });
+
+  it("should do nothing if null include is sent", async () => {
+    const inputManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+    files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+    const expectedManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+      files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+
+    await copyIncludedFiles(inputManifest, null, null);
+
+    expect(inputManifest).to.deep.equal(expectedManifest);
+  });
+
```

Comment:
Consider using context, to capture that *dll.config can be valid/invalid (https://mochajs.org/#bdd):